### ISSUE TITLE
P2P workflow and UI  issues

### ISSUE THREAD

**Issue by madrinekariuki** [opensrp/fhircore#2277]
App: https://github.com/opensrp/fhircore/releases/tag/v0.2.3-rc1
G60 3GB RAM 64GB ROM android 9 -- `CHA`
G60 3GB RAM 64GB ROM android 9 -- `CHSS`
**Describe the bug**
Issues on the P2P workflow and UI

**To Reproduce**
- login to one device with `CHA`
- login to another device with `CHSS` 
- ensure sync is completed on both devices.
- add some data on the `CHA` device e.g add a HH and register members and enroll them to a careplan
- initialize the p2p transfer.
- after its complete confirm you are able to see your transferred data on CHSS 
- repeat the process to test subsequent transfers

Observations:
- After data transfer, the Household data is not opening on chss. It is a blank page. (send data via p2p -> on click on `chss` and open the `CHAs` and open the household data )

- missing below screen in p2p. when you click on pair, it immediately starts sending data without the option to either pair or decline
![image](https://user-images.githubusercontent.com/84722169/234187089-83f0841b-08cc-42d6-9bd3-d4b0dfb0001b.png)

- one can only pair from the sender device.

- when sending subsequent times, the percentage is exceeding 100% see video below.
https://user-images.githubusercontent.com/84722169/234009864-e7dd931f-6538-49d0-98a0-f5e6618e3cc5.mp4

issue reported [here](https://github.com/opensrp/fhircore/issues/1729#issuecomment-1520198859) as well

**Comment by Rkareko** [comment]
> missing below screen in p2p. when you click on pair, it immediately starts sending data without the option to either pair or decline

concerning the above,  the `accept`  / `decline`  functionality is handled by the android OS and we have not yet found a way to override it.

**Comment by pld** [comment]
@madrinekariuki can you break these out into individual issues?

**Comment by ekigamba** [comment]
@madrinekariuki I can't seem to replicate some of these issues on app/debug and therefore this might be an issue replicable only by a specific number of steps or specific instance. Could you tell me which server this was using? Could you also tell me what you added before sending in the video https://user-images.githubusercontent.com/84722169/234009864-e7dd931f-6538-49d0-98a0-f5e6618e3cc5.mp4 .

I'll retest again using the CHA and CHSS devices.

**Comment by madrinekariuki** [comment]
@ekigamba 
Please confirm the specific issues. I can write the steps and add a video

 _Could you tell me which server this was using?_   -- This was using the preview server
 _Could you also tell me what you added before sending in the video?_   -- Currently do not have the specific details of the added data for this. I have added a lot of test data to the app it'd be easier for me to give you fresh data.

**Comment by ekigamba** [comment]
> Could you also tell me what you added before sending in the video? -- Currently do not have the specific details of the added data for this. I have added a lot of test data to the app it'd be easier for me to give you fresh data.

Yes, fresh data is OK and steps so that I can try to replicate this issue.

It might be that there's a specific resource type that has an issue during sending or counting to give the total tally and this will help me narrow down on this

**Comment by madrinekariuki** [comment]
@AnnieMungai 

https://github.com/opensrp/fhircore/releases/tag/v0.2.4-rc3

This passes QA

- All the required workflows are present.

![image](https://github.com/opensrp/fhircore/assets/84722169/1d370d12-5544-4b57-9a8e-d801dcd42b4e)
